python/ray/train/v2/_internal/execution/scaling_policy/fixed.py

```diff
@@ -12,6 +12,8 @@
 
 class FixedScalingPolicy(ScalingPolicy):
     def make_decision_for_non_running_worker_group(self) -> ScalingDecision:
+        if self.scaling_config.num_workers is None:
```

Comment:
alternatively, we can set num_workers=1 if there's no scaling config

Reply:
reverted since this is already fixed https://github.com/ray-project/ray/pull/52079#issuecomment-2787392018